### PR TITLE
Create a doc for updating the CMS

### LINKED ISSUE #1073: How to update to new version?
**- Do you want to request a *feature* or report a *bug*?**

Feature

**- What is the current behavior?**

There is no documentation explaining how to update to a newer version of Netlify CMS.

**- If the current behavior is a bug, please provide the steps to reproduce.**

**- What is the expected behavior?**

A documentation should be created explaining how to update to a newer version of Netlify CMS.

**- Please mention your CMS, node.js, and operating system version.**

**- Please link or paste your `config.yml` below if applicable.**

### PR DESCRIPTION
**Pull request by ragasirtahk**
**- Summary**

Fixes #1073 by creating a doc page about updating the CMS.

**- Test plan**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 02636271c2ef3ab79e17cd5fe63e305f76d618d8

https://deploy-preview-1076--cms-demo.netlify.com

**Comment by ragasirtahk** [comment]
I don't know the reason why it failed.

**Comment by tech4him1** [comment]
Build is not your fault, there is an issue with Yarn right now.

**Comment by ragasirtahk** [comment]
Is it possible to rerun the test build?

**Comment by tech4him1** [comment]
@ragasirtahk Reset. I also updated the title so that it was consistent with the other docs pages (good with you?).

File: website/site/content/docs/updating-the-cms.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Updating the CMS
+position: 24
+---
+
+# How updates to Netlify CMS are done
```
Comment (tech4him1) [inline website/site/content/docs/updating-the-cms.md]: I think that this should be the same as the page title: "Updating the CMS".

Comment (ragasirtahk) [reply website/site/content/docs/updating-the-cms.md]: Done

**Comment by ragasirtahk** [comment]
The preview shows the earlier version of heading, idk why.

**Comment by verythorough** [comment]
@ragasirtahk I recommend renaming it "CMS Version Updates," because "Updating the CMS" might imply to some that the page is about updating content in the CMS.

Also, I would move the position to go after the pages around config and setup, since you're not likely to need to update the version on your first install. I'd say between Custom Previews and Architecture, so `position: 60` should work.

**Comment by erquhart** [comment]
I think installation and upgrading should be documented together, but since our current setup doesn't really have a condensed "installation" guide, I'll second @verythorough's suggestion.

For title, I'd say "Updating the CMS Version", but I don't have a really strong opinion on it.

**Comment by ragasirtahk** [comment]
Renamed to "Updating the CMS Version". 
Set position : 60.

**Comment by verythorough** [comment]
That reminds me! We have an issue for the installation doc: 
https://github.com/netlify/netlify-cms/issues/731

I'm fine merging this doc separately for now, but @ragasirtahk if you'd like to work on closing #731, that would be awesome, too.

For the title, I'd prefer "Update the CMS Version" for two reasons: 1) it matches the other titles in the sidebar, and 2) it's a little less likely to wrap in the sidebar.

**Comment by ragasirtahk** [comment]
Changed to "Update the CMS Version". 

As of now merge this doc separately. My exams start from tomorrow, I will start work on #731 in the last week of this month.

**Comment by ragasirtahk** [comment]
screenshot_20180208-170547
This seems a bug in the doc website.

**Comment by ragasirtahk** [comment]
I have a suggestion. The current "Hugo Site Starter" on https://www.netlifycms.org/docs/start-with-a-template/ uses https://github.com/netlify-templates/one-click-hugo-cms .

The one click Hugo cms repository uses "Victor Hugo" and Netlify CMS which has a bit different structure from the original "Hugo" structure. 

I think the title should be changed to "Victor Hugo Site Starter" and a new template should be created for "Hugo Site Starter". This new template will only use pure "Hugo" and "Netlify CMS". 

Does it seems good to you? 
I can work on the new template and complete it by first week of march. 

Should I create a new issue for this if you agree on my viewpoint?

**Comment by verythorough** [comment]
@ragasirtahk LGTM. Thanks for your contributions!
Could you put up your last two comments as new issues?

**Comment by ragasirtahk** [comment]
Created new issues for them. #1090 and #1091

**Pull request merged**